compiler_opt/rl/local_data_collector.py

```diff
@@ -76,15 +77,13 @@ def _join_pending_jobs(self):
     logging.info('Waiting for pending work from last iteration took %f',
                  time.time() - t1)
 
-  def _schedule_jobs(
-      self, policy_path, sampled_file_paths
-  ) -> List[worker.WorkerFuture[compilation_runner.CompilationResult]]:
```

Comment:
accidental deletion?

Reply:
looks that way - I'll fix it (thanks for finding it!)